printer_mods/README.md

```diff
@@ -85,6 +85,7 @@ like so:
  ByteChild | [ESP32 DIN Rail Bracket](./ByteChild/ESP32_DIN_Rail_Bracket) | DIN rail mounting bracket for an ESP32 | :x: | :grey_question: | :heavy_check_mark: | :x: |
  || [2-Channel Relay DIN Rail Bracket](./ByteChild/2_Channel_Relay_DIN_Rail_Bracket) | DIN Rail mounting bracket for an 2-Channel-Relay | :x: | :grey_question: | :heavy_check_mark: | :x: |
 |bythorsthunder|[MGN9 Backers Idlers](./bythorsthunder/MGN9_Backers)|New idler uppers for MGN9 Extrusion backers |:x:|:x:|:heavy_check_mark:|:x:|:x:|
+||[Stealthburner Rapido UHF](./bythorsthunder/Stealthburner_Rapido_UHF)|Stealthburner mod for Rapido UHF |:x:|:x:|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|
```

Comment:
You've got a case-mismatch in the folder name here:

Link to unknown file: `bythorsthunder/Stealthburner_Rapido_UHF`. Did you mean `bythorsthunder/Stealthburner_Rapido_Uhf`

Reply:
Fixed.   Thank you!